sdks/permit2-sdk/src/constants.ts

```diff
@@ -1,7 +1,17 @@
 import { BigNumber } from '@ethersproject/bignumber'
 
+// @deprecated please use permit2Address(chainId: number)
 export const PERMIT2_ADDRESS = '0x000000000022D473030F116dDEE9F6B43aC78BA3'
 
+export function permit2Address(chainId?: number): string {
+  switch (chainId) {
+    case 324:
+      return '0x0000000000225e31d15943971f47ad3022f714fa'
```

Comment:
nit: should it be checksummed? the other one is.
The check sum version is `0x0000000000225e31D15943971F47aD3022F714Fa`

Reply:
Sure, can make it checksummed, although I think functionally they are equivalent, because eth addresses are case insensitive.

Reply:
checksummed